frontend/src/Editor/Inspector/Components/FilePicker.jsx

```diff
@@ -30,7 +30,7 @@ export const FilePicker = ({ componentMeta, darkMode, ...restProps }) => {
     const conditionalOptions = [{ name: 'parseFileType', condition: parseContent }];
```

Comment:
Can we check the condition here: 
`const conditionalOptions = [{ name: 'parseFileType', condition: parseContent === '{{true}}' }];`